central/graphql/resolvers/nodes.go

```diff
@@ -385,31 +439,61 @@ func (resolver *nodeResolver) Vulns(ctx context.Context, args PaginatedQuery) ([
 	}), PaginatedQuery{Query: &query, Pagination: args.Pagination})
 }
 
+// Vulnerabilities returns all of the vulnerabilities in the node.
+func (resolver *nodeResolver) Vulnerabilities(ctx context.Context, args PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
+	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.Nodes, "Vulnerabilities")
+	if err := readNodes(ctx); err != nil {
+		return nil, err
+	}
+
+	if !features.PostgresDatastore.Enabled() {
+		query := search.AddRawQueriesAsConjunction(args.String(), resolver.getNodeRawQuery())
```

Comment:
conjunction query is not required since you are adding scope below. `scoped.Scope` ensures that all the queries are in the specified scope.

Reply:
Removed the conjunction query.